pallets/tinlake-investor-pool/src/lib.rs

```diff
@@ -443,6 +460,8 @@ pub mod pallet {
 					max_reserve,
 					available_reserve: Zero::zero(),
 					total_reserve: Zero::zero(),
+					min_epoch_time: T::DefaultMinEpochTime::get(),
```

Comment:
We could also add this as an argument. However, I think this is an area where it's good to enforce some conventions (i.e. a default min epoch time and max nav age per runtime), but still allow pool owners to change them if needed.